frontend/src/constants/theme.ts

```diff
@@ -52,6 +52,7 @@ const themeColors = {
 	gamboge: '#D89614',
 	bckgGrey: '#1d1d1d',
 	lightBlue: '#177ddc',
+	lightGrey: '#acacac',
```

Comment:
grey is already present in color from antd/colors

Reply:
Okay